deployment/local-jvm/bash-scripts/startServices.sh

```diff
@@ -9,7 +9,7 @@ if [ -d $DIR ]; then
   # Start all the services using the service manager from Palisade-services
   if [ -f $FILE ]; then
     java -jar -Dspring.profiles.active=discovery $FILE --run
-    java -jar -Dspring.profiles.active=services,debug,eureka $FILE --run
+    java -jar -Dspring.profiles.active=example,debug $FILE --run
```

Comment:
Is there a reason to have example and services profiles in the services repo? I cant imagine needing two separate profiles.

Reply:
The example profile is there to specifically pick up the application-example yaml from within the example-library which contains the example data to be cached. There may be extra fields in the loaded data (for example the training courses for a User).

Reply:
Awaiting some back-and-forth on services PR

Reply:
Awaiting some back-and-forth on services PR